test/extended/networking.sh

```diff
@@ -318,6 +318,13 @@ else
 
   os::util::environment::setup_tmpdir_vars "test-extended/networking"
 
+  # Allow setting $JUNIT_REPORT to toggle output behavior
+  if [[ -n "${JUNIT_REPORT:-}" ]]; then
+    export JUNIT_REPORT_OUTPUT="${LOG_DIR}/raw_test_output.log"
```

Comment:
I think the raw output should vary by plugin, as the regular output already does or the output of only one of the plugins will be saved:

https://github.com/openshift/origin/blob/master/test/extended/networking.sh#L167